stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
+items:
+  - metadata:
+      labels:
+        name: weave-flux
+      name: weave-flux
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-flux-agent
+        app: weave-flux
+        weave-cloud-component: flux
+        weave-flux-component: agent
+      name: weave-flux-agent
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    spec:
+      template:
+        metadata:
+          labels:
+            name: weave-flux-agent
+            app: weave-flux
+            weave-cloud-component: flux
+            weave-flux-component: agent
+        spec:
+          containers:
+            - name: agent
+              image: 'quay.io/weaveworks/fluxd:0.1.0'
```

Comment:
We prefer to use our upstream manifest (maintained as an API see comment below regarding `update.sh`).
I am happy to apply some minimal transformation to the version that would get check-in here, but I would refrain from using templates for names of resources and would prefer to keep our well-known names instead.

> ...so that they can be changed.

Could you please explain what is the use benefit of that?